src/components/stages/index.js

```diff
@@ -667,8 +683,11 @@ const Stages = forwardRef(
                         }
                         label="Unhealthy only"
                         onChange={() =>
-                          setSignalExpandOption(
-                            (seo) => seo ^ SIGNAL_EXPAND.UNHEALTHY_ONLY
+                          setSignalExpandOption((seo) =>
+                            signalChangeHandler(
+                              seo,
+                              SIGNAL_EXPAND.UNHEALTHY_ONLY
+                            )
```

Comment:
Prefer setting nerdlet state here, and have a `useEffect` to set `signalExpandOption`